Re-added a deleted image and removed a broken link

GBKS opened the pull request:
The address page in the glossary was referencing an image from the request page. That image was deleted in the last update to the page, so I restored it.

Similar thing with the link. Contacts linked to a section on the request page that was removed.

Bosch-0 commented on guide/glossary/address.md:
```diff
@@ -84,8 +84,8 @@ Since "legacy" addresses are still in use, some older applications have yet to u
-   image = "/assets/images/guide/payments/request/single-use-address-type.png"
-   retina = "/assets/images/guide/payments/request/single-use-address-type@2x.png"
+   image = "/assets/images/guide/glossary/address/single-use-address-type.png"
+   retina = "/assets/images/guide/glossary/address/single-use-address-type@2x.png"
```
```suggestion
   retina = "/assets/images/guide/payments/request/single-use-payment-request-settings-address@2x.png"
```

It was deleted on purpose but I was suppose to replace it with this updated image - sorry about that!

GBKS replied:
OK. I'll update this PR and remove the image I re-added.

GBKS commented:
@Bosch-0 just addressed your feedback. Overall, I am not a big fan of one page referencing images in the image folder of another page. Things can get easily out-of-sync as those relationships are not obvious. How should we treat this here?

Bosch-0 commented:
Personally I don't think we need images in the glossary, I'm fine with it just being text based. Can link from the glossary to this section in the requesting page though

pavlenex commented:
@GBKS There's a conflict that needs to be resolved.

GBKS commented:
@pavlenex fixed the merge issue.

@Bosch-0 this is just a bug fix PR. Whether images go in the glossary or not is a separate discussion. My question was just about whether to enforce the rule that image files always have to be in the folder that matches the pages URL structure.
